lib/cldr/export/data/calendars/gregorian.rb

```diff
@@ -92,7 +92,10 @@ def eras
                 path = "#{base_path}/#{name}/*"
                 key  = name.gsub('era', '').gsub(/s$/, '').downcase.to_sym
                 result[key] = select(path).inject({}) do |ret, node|
-                  type = node.attribute('type').value.to_i rescue 0
+                  # TODO: Actually resolve alias nodes
+                  # https://github.com/ruby-i18n/ruby-cldr/issues/78
+                  next {} if node.name == "alias"
```

Comment:
The `root` locale contains `<alias>` nodes for calendar eras:

* [`dates/calendars/calendar[8]/eras/eraNames/*`](https://github.com/unicode-org/cldr-staging/blob/f20483bff67e44a1326129d59563a52657b0bb04/production/common/main/root.xml#L1264-L1266)
* [`dates/calendars/calendar[8]/eras/eraNarrow/*`](https://github.com/unicode-org/cldr-staging/blob/f20483bff67e44a1326129d59563a52657b0bb04/production/common/main/root.xml#L1271-L1273)

These are incorrectly getting output as:

```yaml
eras:
  abbr:
    0: BCE
    1: CE
  name:
    0: ''
  narrow:
    0: ''
```

Until #78 is addressed, I figure it is better not to output any data, rather than incorrect data.